bip-musig2.mediawiki

```diff
@@ -247,7 +247,7 @@ Input:
 '''''GetSecondKey(pk<sub>1..u</sub>)''''':
 * For ''j = 1 .. u'':
 ** If ''pk<sub>j</sub> &ne; pk<sub>1</sub>'':
-*** Return ''pk<sub>j</sub>''
+*** Return ''bytes(pk<sub>j</sub>)''
```

Comment:
`pk<sub>j</sub>` is already a byte array

Reply:
Right, my bad, I didn't chase it far up the stack enough :) will remove.

Reply:
Fixed in https://github.com/jonasnick/bips/pull/23/commits/789a909dd1ea0b4818e9e090b33066e44dd99a33